examples/add_labels_with_properties.py

```diff
@@ -0,0 +1,20 @@
+"""
+Display a labels layer with classes for each label, including the background
+"""
+
+import napari
+import numpy as np
+from skimage import data
+from scipy import ndimage as ndi
+from napari.layers import Labels
+
+blobs = data.binary_blobs(length=128, volume_fraction=0.1, n_dim=2)
+labelled, num_labels = ndi.label(blobs)
+class_dict = {
+    'class': ["Background"]
+    + ["Class " + str(i + 1) for i in range(num_labels)]
+}
```

Comment:
This example is sort of uninteresting, because each label is its own class. What I would do here is divide blobs into large and small and then map to just two classes, "small" and "large" or "simple" and "compound", for example

Reply:
Yeah I thought so too but I couldn't think of anything more interesting... lol

Reply:
We just commented at the same time, see my suggestion below about using the coins like we do in `add_labels.py` (I also just like looking at the coins) :-)